### PR TITLE
add release 1.37 post

### PR DESCRIPTION
**Pull request by solvingj**

### CODE REVIEW

**Review by memsharded — APPROVED** [review]

**Review by danimtb — APPROVED** [review]

**Review by danimtb — CHANGES_REQUESTED** [review]
We need to add a highlight in this post about the new `conancenter` remote. This is important as this is no longer a BETA remote (https://blog.conan.io/2021/04/23/New-Conan-Center-url.html) but actually the official remote for conan center with recipes only coming from conan-center-index (https://github.com/conan-io/conan-center-index).

Also, the old `conan-center` (bintray) remote will stop syncing packages from `conancenter` remote on June 1st. Eventually, `conan-center` remote will be removed as default remote in the conan client.

File: _posts/2021-06-10-New-conan-release-1-37.markdown [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+There are essentially two ways to create a Conan package:
+
+* `conan create` : compiles sources into binaries and then packages them
+* `conan export-pkg` : takes precompiled binaries and packages them
+
+One of Conan's marquee features is the unique ability to pass `--build` to the
+`conan install` command and have Conan re-build some or all dependencies in the
+graph from source.  Unfortunately, for any packages in the dependency graph
+which were created using `conan export-pkg` commnd, rebuilding from source is
```
Comment (jgsogo) [inline _posts/2021-06-10-New-conan-release-1-37.markdown:130]:
```suggestion
which were created using `conan export-pkg` command, rebuilding from source is
```

**Review by jgsogo — APPROVED** [review]

**Comment by solvingj** [comment]
@danimtb , good catch.  i added a section right at the top. please review.

File: _posts/2021-06-10-New-conan-release-1-37.markdown [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+* `https://center.conan.io`
+* `https://conan.bintray.com`
+
+It's important to understand that these URL's point to two separate Conan
+repositories, and they have the following differences:
+
+### `https://conan.bintray.com`
+
+This repository contains copies of all Conan packages, new and legacy (by
+"legacy" we mean packages which predate ConanCenterIndex.) Currently, copies of
```
Comment (jgsogo) [inline _posts/2021-06-10-New-conan-release-1-37.markdown:34]:
```suggestion
"legacy" we mean packages which predate ConanCenterIndex). Currently, copies of
```

File: _posts/2021-06-10-New-conan-release-1-37.markdown [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+This repository only contains copies of all Conan packages which have been built
+from ConanCenterIndex (no legacy packages). Currently, copies of all new package
+builds and binaries are uploaded here.  On **July 1st**, this repository will
+become the **ONLY** place where new packages are uploaded. This is why we have
+added it to the client in the first position.
+
+### Long-term deprecation: `https://conan.bintray.com`
+
+We have not yet set a date for the removal of the conan.bintray.com URL from the
+default list of remotes, nor the physical deletion of the repository and it's
```
Comment (jgsogo) [inline _posts/2021-06-10-New-conan-release-1-37.markdown:50]:
```suggestion
default list of remotes, nor the physical deletion of the repository and its
```

**Review by danimtb — APPROVED** [review]

**Pull request merged**